src/util/subprocess.hpp

```diff
@@ -1549,7 +1256,7 @@ inline void Popen::execute_process() noexcept(false)
                             NULL,         // primary thread security attributes
                             TRUE,         // handles are inherited
                             creation_flags,	// creation flags
-                            environment_string_table_ptr,  // use provided environment
+                            NULL,         // use environment of the calling process
```

Comment:
nit: Maybe a wording more consistent with other comment around:
```suggestion
                            NULL,         // use parent's environment
```